src/main/java/studos/logic/I18N.java

```diff
@@ -61,8 +61,17 @@ private static boolean languageReadFromConfig() {
      * @return List of Locale objects.
      */
     public static List<Locale> getSupportedLocales() {
-        return new ArrayList<>(Arrays.asList(Locale.forLanguageTag("en"),
-         Locale.forLanguageTag("pl")));
+        return new ArrayList<>(Arrays.asList(Locale.forLanguageTag("pl-PL"),
+        Locale.forLanguageTag("en-US")));
+    }
```

Comment:
We should work on system language not system region.

In situation where someone will have english system with (for example) polish language the application will run default in english mode which is bug.